src/main/java/com/kiwigrid/k8s/helm/tasks/HelmRepoSyncTask.java

```
@@ -161,7 +160,7 @@ private void addRepo(HelmRepository repo) {
 					"--password=" + repo.getPassword()
 			);
 		} else {
-			helmExec(getProject(), this,
+			helmExecSuccess(getProject(), this,
```

this change is what tricked me previously.

suppose that I have this configuration:

```
...
repositories {
        "hub" {
            url "https://hub.net/chartrepo/snap/charts"
```

Now this url : `https://hub.net/chartrepo/snap/charts` is invalid and the correct one is suppose to be : `https://hub.net/chartrepo/snap`

This would throw an `Exception` internally (in our case), but because `helmExec` was used, this would fail silently, and ultimately this would "die" with `FileNotFoundException` saying that `repositories.yaml` is not found. 

This is indeed the case, but not because it is not found really, but because the repo could not be reached, because he URL was wrong.

Changing this to `helmExecSuccess` would report something like:

```
Execution failed for task ':helmRepoSync'.
> Unexpected failed execution:
  Error: looks like "https://hub.net/chartrepo/snap/charts" is not a valid chart repository or cannot be reached: failed to fetch https://hub.net/chartrepo/snap/charts/index.yaml : 500 Internal Server Error
``` 

which of course now makes perfect sense.